src/components/input-style/input-style.scss

```diff
@@ -225,6 +225,18 @@ $m-input-style-base-height: $m-line-height + em;
 
     &__right-content {
         display: flex;
+
+        .m-moneyfield & {
```

Comment:
J'sais pas trop comment ça été fait mais c'est bizare d'avoir des .money-field dans le input-style.
@Atiomi C'est voulu que ça soit comme ça? Pcq je vois que ça pas été introduit purement par Caro// reserve space for icon
.m-datepicker .m-input-style__label,
.m-timepicker .m-input-style__label,
.m-moneyfield .m-input-style__label,
.m--is-type-search .m-input-style__label,
.m--is-type-search .m-input-style__adjust-width-auto {
    padding-right: $m-input-style--icon-button-size + $m-input-style--padding;
}

Reply:
J'aurais pu mettre le style dans la feuille de style de m-moneyfield mais comme il est scopé, il aurait fallu que je mette un deep, alors j'ai préféré le mettre directement dans le style du input-style, ce n'est peut-être pas la meilleure façon de faire, je peux le changer de place s'il faut.

Reply:
@Atiomi, @fredduguay avez-vous une opinion ? Est-ce qu'on merge comme ça ou Caro doit faire des changements ?

Reply:
C'est sur que ce n'est pas l'idéal, ça prendrait une mécanique vue pour mieux gérer ces cas, mais en vrai c'est tout le input-style qui faudrait revoir.
D'un point de vue css je préfère regrouper le code dans le fichier des éléments modifiés, alors oui ça me convient.

Reply:
Je suis d'avis qu'on devrait éviter de mettre des styles de moneyfield directement dans le css de input style. Je comprend que le scoping rend le processus plus compliqué mais il faudrait voir si on peut bonifier input style de façon à répondre au besoin.

Reply:
Je suis d'avis qu'on devrait éviter de mettre des styles de moneyfield directement dans le css de input style. Je comprend que le scoping rend le processus plus compliqué mais il faudrait voir si on peut bonifier input style de façon à répondre au besoin.

Reply:
Je crois que l'idéal est de conserver les styles du `m-moneyfield` dans le `m-input-style`, car les styles sont dépendants du balisage du `m-input-style`. Si les styles sont appliqués dans le `m-moneyfield`  avec le deep et que le balisage HTML du `input-style` change, les styles risquent de ne plus s'afficher correctement sans être notifié.

L'autre option serait d'ajouter une prop skin au `input-style`.

Reply:
C'est une discussion qui sort du scope de ce PR. Je propose de prendre en note ce problème et d'en discuter plus largement pour une version ultérieure.